### ISSUE TITLE
Some Titles are too long

### ISSUE THREAD

**Issue by ghost** [SandwichFox/MBCord#29]
**Describe the feature/change you'd like**
Instead of throwing an error if the title is too long, make it cut down or just don't add an Title.

**Additional context**
![grafik](https://user-images.githubusercontent.com/83706294/205450113-58502f74-5930-4473-9cf0-3c7ade3ec3e4.png)
This happened because the title is in German and its too long.
Example:
```
Classroom of the Elite - S1:E6 - Es gibt zwei Arten von Lügen: Eine, welche Tatsachen betrifft und damit in die Vergangenheit zurückweist, und eine andere, bei welcher es sich um Versprechungen für die Zukunft handelt.
```
That's to long and it will throw an error, which won't display the Discord Presence.

Excuse me for my lack of english

**Comment by SandwichFox** [comment]
Weird.. Ich werds mir mal anschauen.

**Comment by SandwichFox** [comment]
Fixed in Next Release 3.0.7
